sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -1514,6 +1531,42 @@ def process(self, element, *side_inputs):
     yield (self.destination(element, *side_inputs), element)
 
 
+def beam_row_from_dict(row: dict, schema):
```

Comment:
Let's make sure this util is extensively covered by unit testing for various types etc.